docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
       - "SMARTHOST_ALIASES=${SMARTHOST_ALIASES}"
 
   redis:
-    image: redis:7.2
+    image: eqalpha/keydb:x86_64_v6.3.4
```

Comment:
This image is x64 only while the Redis image is ARM as well. Would it be possible to remove the arch modifier here?

Reply:
Note that unless MISP changes the default key-value storage, we should only document this alternative in the README.md rather than modify the `docker-compose.yml` file

Reply:
Fine by me. All their images are arch-specific except eqalpha/keydb:latest which is both for arm64 and x86_64.

Reply:
We should also reference https://github.com/valkey-io/valkey if a docker image is available.